internal/docgen/main.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func main() {
-	f, err := os.Create("./docs/reference/cli-reference.md")
```

Comment:
we should add a redirect for anything pointing to cli-reference to cli

Reply:
#3432